packages/api-explorer/src/utils/path.ts

```diff
@@ -128,3 +132,35 @@ export const getSpecKey = (location: HLocation | Location): string | null => {
   }
   return match?.groups?.specKey || null
 }
+
+/**
+ * Gets the scene type of the current page
+ * @param location browser location
+ * @returns string representing the scene type
+ */
+export const getSceneType = (location: HLocation | Location) => {
+  const match = matchPath<{ tabType: string }>(location.pathname, {
+    path: '/:specKey/:tabType',
+  })
+  return match ? match!.params.tabType : ''
+}
+
+/**
+ * Confirms if filter is valid for the page scene type
+ * @param location browser location
+ * @param filter filter tag for page
+ */
+export const isValidFilter = (
+  location: HLocation | Location,
```

Comment:
Given you preferred having the string pathname for the method `getSceneType`, would you prefer changing this from a location object to a string?